src/trigger.js

```diff
@@ -0,0 +1,189 @@
+try {
+  var m = require("mithril");
+} catch (e){
+}
+
+const app = {};
+
+app.view = function(vnode) {
+  const state = vnode.state;
+
+  const matchSearchKey = (project) => {
+    const keys = state.search_key.split(" ");
+    return keys.every((key) => {
+      const r = new RegExp(key, "i");
+      return project.path_with_namespace.match(r);
+    });
+  };
+
+  const projectEvents = (project) => {
```

Comment:
I think that `const projectEvents` can be erased because `project.events` is no used.

Reply:
done